docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     container_name: akd-test-db
     platform: linux/x86_64
     image: mysql
-    command: --default-authentication-plugin=mysql_native_password
+    command: --mysql-native-password=ON
```

Comment:
Hmm, what's the reason for this change? Just curious

Reply:
This is due to MySQL 8.4 docker image made a breaking change in the way it takes the command line parameters. Without this, the CI for mysql test will timeout due to hanging around this error 
```
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
```

More context is https://stackoverflow.com/questions/78445419/unknown-variable-default-authentication-plugin-mysql-native-password